.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: ruby
-sudo: false
+cache: bundler
```

Comment:
`sudo: false` is deprecated and removed from Travis. `cache: bundler` should speed up test runs.

Reply:
thanks for the comment. saves us from tracking it down